editor/sidebar/post-settings/index.js

```diff
@@ -18,6 +18,7 @@ import PostTaxonomies from '../post-taxonomies';
 import FeaturedImage from '../featured-image';
 import DiscussionPanel from '../discussion-panel';
 import LastRevision from '../last-revision';
+import TableOfContent from '../table-of-content';
```

Comment:
I've not seen this referred with the singular form of content, always "Table of Contents"

https://en.wikipedia.org/wiki/Table_of_contents